apps/microtvm/reference-vm/zephyr/rebuild-tvm.sh

```diff
@@ -18,6 +18,13 @@
 
 set -e
 
+# Get number of cores for build
+if [ -n "${TVM_CI_NUM_CORES}" ]; then
+  num_cores=${TVM_CI_NUM_CORES}
+else
+  num_cores=4
```

Comment:
why 4? can you add comment?

Reply:
it should be 2 since that is the default value in Vagrantfile. I changed it.